src/config/dynamic-theme-fixes.config

```diff
@@ -32495,6 +32495,7 @@ www.mozilla.org
 INVERT
 img.sidebar-icon
 h1.c-logo
+.m24-c-navigation-logo-image
```

Comment:
I checked and for me it seems the selector is `.c-navigation-logo-image`

Can you check again?

Reply:
Yes, thanks.  It's different for me.  Which browser did you use?  I think we will need to add both classes.

Reply:
![obraz](https://github.com/user-attachments/assets/e1f9ff69-e828-4a50-981d-7f097d54cb66)
Width: 2560px as my monitor is 2560x1080 and I use Firefox in Full Screen.

But for me the main page opens as /pl/ when I changed to En-US I see your class.

So yeah... I think it will be better to add both classes. 😉

Reply:
Done! Thank you!